cmd/warden/app/options/flags.go

```diff
@@ -142,5 +142,15 @@ var (
 			Value:       true,
 			Destination: &WardenOpts.GenericWardenOpts.InMemberCluster,
 		},
+		&cli.StringFlag{
+			Name:        "namespace",
+			Value:       "kubecube-system",
+			Destination: &WardenOpts.GenericWardenOpts.Namespace,
```

Comment:
There is no need to specify namespace for warden, since all components are supposed to running under `kubecube-system` cc @zhujf1989